app/javascript/src/require_communication_preference.js

```diff
@@ -41,8 +42,13 @@ $('document').ready(() => {
       displayPopUpIfPreferencesIsInvalid(receiveEmail, receiveSMS, true)
     })
   }
+
+  const smsToggle = document.getElementById(SMS_TOGGLE_CLASS)
```

Comment:
`document.getElementById(CLASS)` throws null errors attempting to find the element on any given site page (inspect element console error). `$(.${CLASS})[0]` should fix this while also avoiding weird occurrences like allowing an array of values to pass.